pkg/server/claims.go

```diff
@@ -157,6 +159,18 @@ func parsePrivateClaims(oidcToken jwt.Token) (*oidcClaims, error) {
 	return &claims, nil
 }
 
+func extractRepository(oidcToken jwt.Token) (string, error) {
+	if oidcToken.Issuer() == config.GitHubIssuer {
+		return requiredClaim(oidcToken, "repository")
+	}
+
+	if len(oidcToken.Audience()) != 1 {
+		return "", fmt.Errorf("non-github OIDC token's audience field should have exactly one entry of a repository containing a minty config")
+	}
+	scopeRepository, _ := strings.CutPrefix(oidcToken.Audience()[0], "https://github.com/")
```

Comment:
I might be wrong, but could there be some possibility that the `Audience` doesn't have `https://github.com/` as prefix? Should we handle case like that?

Reply:
The supported formats are `https://github.com/org/repo` or simply `org/repo` and this code handles both by trimming the github URL prefix off if it is present.

Reply:
Thanks!